icc-x-api/icc-contact-x-api.ts

```diff
@@ -565,7 +565,10 @@ export class IccContactXApi extends iccContactApi {
                                   console.log("Cannot decrypt service", svc.id)
                                   resolve(null)
                                 }
-                              )
+                              ).catch(e => {
+                                console.log("Cannot decrypt service", svc.id);
+                                return null;
```

Comment:
- `return null;` probably unnecessary as the function will return falsy `undefined `anymay.
- avoid console.log. Prefer `console.warn()` or LoggerService.